### PR TITLE
fix: miscellaneous backend fixes

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 594f313cfc30d9a5271aa93d593b71a5607604f9

https://deploy-preview-1568--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 594f313cfc30d9a5271aa93d593b71a5607604f9

https://deploy-preview-1568--cms-demo.netlify.com

File: packages/netlify-cms-backend-bitbucket/src/API.js [hunk 1]
```diff
@@ -118,10 +118,15 @@ export default class API {
     return this.processFiles(entries);
   };
 
-  uploadBlob = async item => {
+  uploadBlob = async (item, { commitMessage, branch = this.branch, author = this.commitAuthor } = {}) => {
```
[the remark this answers is not in the document]
Comment (tech4him1) [reply packages/netlify-cms-backend-bitbucket/src/API.js]: Good catch, it should be added to the commit.

**Comment by tech4him1** [comment]
@erquhart Updated!

**Pull request merged**